austrakka/components/metadata/funcs.py

```diff
@@ -73,3 +76,15 @@ def list_metadata(group_name: str, out_format: str):
             'groupContext': group_id
         }
     )
+
+@logger_wraps()
+def list_metadata_by_field(group_name: str, field_names: List[str], out_format: str):
+    group_id: str = get_group_by_name(group_name)['groupId']
+    call_get_and_print_table(
+        f'{METADATA_SEARCH_PATH}/{METADATA_BY_FIELD_PATH}',
+        out_format,
+        params={
+            'groupContext': group_id,
+            'fields': field_names
```

Comment:
Will field_names to serialized to a comma-separated string?

Reply:
Actually good question. Yes that's how it serialises. But, I had thought the query expected a format like `&fields=Seq_ID&fields=Date_coll`. This is how it looks in swagger. And according to debug AppInsights, this is how the request is coming in locally.

This code DOES seem to work (deserialised correctly by the .NET FromQuery-based fields parameter). We're calling httpx's `client.get()` with the fields passed in as a comma-separated list... I'm not sure where the magic is happening.

Reply:
Sorry, it is passed in as a list, it _serialises_ in the URL format described above.